src/shared/lib/StoreUtils.ts

```diff
@@ -1219,6 +1219,17 @@ export function groupBySampleId(
     }));
 }
 
+export function mapSampleIdToClinicalData(
+    clinicalDataGroupedBySampleId: Array<{ id:string; clinicalData:ClinicalData[] }>
+) {
+    const sampleIdToClinicalDataMap = _
+        .chain(clinicalDataGroupedBySampleId)
+        .keyBy('id')
+        .mapValues('clinicalData')
```

Comment:
these string shortcuts are not typesafe so we don't use em.  i think what this should be is .mapValues( (o)=>o.cliniclaData).   it's possible typings have gotten better since i last checked but in the past if you mistyped the string, it would not let you know.